### DISCUSSION TITLE
Units of MD22 dataset

### DISCUSSION THREAD

**Discussion by rfhari** [ACEsuit/mace#460] [Q&A] [answered]
Hi, 

I'm trying to replicate the results obtained in this [paper](https://pubs.aip.org/aip/jcp/article/159/4/044118/2904837/Evaluation-of-the-MACE-force-field-architecture), particularly for the Buckyball catcher in the MD22 dataset. I understand that MACE uses energy in eVs and distances in Å. However, I'm not sure if the units in the Buckyball xyz files are aligned with MACE requirements or if I need to modify them. Can you please clarify this?

**Comment by gabor1** [comment]
Load it up in a viewer, eg Ovito, and check the length of carbon-carbon bonds. In Angstroms it should be between 1.2 and 1.4

**Comment by rfhari** [comment]
Thanks for the quick reply! Sorry I meant about the units of energy. Can you please confirm if it is in eV or kcal/mol or something else?

**Reply by ilyes319** [reply, marked as answer]
You need to convert from kcal/mol to eV.
